project/Setup.scala

```diff
@@ -0,0 +1,62 @@
+import sbt.Keys.javacOptions
+import sbt.TaskKey
+import sbtassembly.AssemblyKeys.assemblyMergeStrategy
+import sbtassembly.AssemblyPlugin.autoImport
+import sbtassembly.AssemblyPlugin.autoImport.{MergeStrategy, assembly}
+import sbtassembly.PathList
+
+/*
+ * Copyright 2024 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+object Setup {
+
+  //supported Scala versions
+//  val scala212 = Version.asSemVer("2.12.18") TODO
+//  val scala213 = Version.asSemVer("2.13.11")
+  val scala212 = "2.12.18"
+  val scala213 = "2.13.11"
+
+  val serviceScalaVersion: String = scala213
```

Comment:
Maybe we could align the naming as we sometimes use server and sometimes service. I would suggest to use server everywhere given it's a name of the module.

Reply:
I think, the original idea had been) (discussed bakc than with @lsulak ) that:
server = REST server
service = all parts of the running service (REST server + database)
If it's not consistent somewhere though, definitely wrong.

Reply:
Alright, I comprehend the logic behind it. However, without prior knowledge, it might not be immediately clear. Hence, I would suggest being more descriptive, for instance, using a name like "serverAndDatabaseScalaVersion".